syncserver/src/web/handlers.rs

```diff
@@ -32,6 +33,15 @@ pub async fn get_collections(
     db_pool: DbTransactionPool,
     request: HttpRequest,
 ) -> Result<HttpResponse, ApiError> {
+    let _hashed_fxa_uid = String::from(&meta.user_id.hashed_fxa_uid);
+    let _user_agent = request
+        .headers()
+        .get("USER_AGENT")
```

Comment:
prefer the constant in `actix_web::http::header`
```suggestion
        .get(header::USER_AGENT)
```

Reply:
I was thinking when I wrote this that I should use the enum 👍